tests/testthat/test-tabPanel.R

```diff
@@ -0,0 +1,14 @@
+
+context("tabPanel")
+
+test_that("tabPanelBody", {
```

Comment:
I think you should include a test that verifies the previous syntax continues to work too.

Reply:
Which syntax? `tabPanelBody(someHTML, value = X)`?

Reply:
Yeah